packages/webdriver/src/web-driver.ts

```diff
@@ -42,15 +45,18 @@ export function* connect(driver: WebDriver, options: Options): Operation<void> {
       .over(args => args.concat(['--headless']))
   }
 
-  driver.session = yield request(`${driver.serverURL}/session`, {
+  driver.session = yield post(`${driver.serverURL}/session`, {
     method: 'post',
     body: JSON.stringify({
       capabilities: capabilities.get()
-    })
+    }),
+    headers: {
+      "Content-Type": 'application/json'
+    }
   });
 }
 
-function* request(url: string, init: RequestInit): Operation<WDResponse> {
+function* post(url: string, init: RequestInit): Operation<WDResponse> {
   let response: Response = yield fetch(url, init);
```

Comment:
Since we're always posting JSON, let's have the `post` method always set the content type header, method, and serialize the body:

```ts
let response: Response = yield fetch(url, {
  ...init, 
  method: 'post',
  headers: {
    "Content-Type": "application/json"
  }
});

Reply:
By serialize the body, is this what you meant? I wasn't sure how to type the body.
```
function* post(url: string, body: string|object): Operation<WDResponse> {
  let response: Response = yield fetch(url, {
    body: JSON.stringify(body),
    method: 'post',
    headers: {
      "Content-Type": "application/json"
    }
  });
  // ...
}
```